pipeline/rhos_scripts/Jenkinsfile-quota.groovy

```diff
@@ -10,7 +10,7 @@ node("rhel-8-medium || ceph-qe-ci") {
             checkout(
                 scm: [
                     $class: 'GitSCM',
-                    branches: [[name: "origin/master"]],
+                    branches: [[name: "origin/fixQuota"]],
```

Comment:
are we proceeding with fixQuota instead of master branch?

Reply:
This change would be reverted after unit testing is successful.